stock_available_to_promise_release/models/stock_move.py

```diff
@@ -85,6 +87,25 @@ def _is_unreleaseable(self):
             and self.rule_id.available_to_promise_defer_pull
         )
 
+    def _in_progress_for_unrelease(self) -> bool:
+        """
+        This method will return True if a move :
+
+        - is not done or canceled
+        - and if the move's picking is not printed
+        - and the move has no quantity done.
+
+        """
+        moves = self.filtered(lambda m: m.state not in ("done", "cancel"))
+        if not moves:
+            return False
+        picking_ids = moves.mapped("picking_id")
+        if picking_ids.filtered("printed"):
+            return True
```

Comment:
@jbaudoux I'm wodering if I should return the recordset in order to be able to get a comprehensive message to user (with picking names)

Reply:
Done